mpi-serial build fail on Anvil

I am working on developments for ACME’s single column model (SCM).  I recently updated my code from CIME5 to the most recent master (CIME5.2).  The new code appears to compile/run fine on Edison.  However, on Anvil it gets stuck early on during the build (it worked in my CIME5 version).  The model builds fine when I compile the full model on Anvil, but not SCM as it appears to be a mpi-serial issue.  Setting up with an X compset (-res f19_g16 -compset X -mpilib mpi-serial) the model build fails with the same message.   

The tail of my error message is below:

gmake -f /home/bogensch/ACME_SCM_sims/Cases/run_acme_scm_RICO_cons_droplet.mpiserial.001/run_acme_scm_RICO_cons_droplet.mpiserial.001/Tools/Makefile /home/bogensch/ACME_SCM_sims/Cases//run_acme_scm_RICO_cons_droplet.mpiserial.001/build/intel/mpi-serial/nodebug/nothreads/mct/mpi-serial/Makefile.conf MODEL=mpi-serial
cat: Filepath: No such file or directory
/home/bogensch/ACME_SCM_sims/Cases/run_acme_scm_RICO_cons_droplet.mpiserial.001/run_acme_scm_RICO_cons_droplet.mpiserial.001/Tools/Makefile:229: *** commands commence before first target.  Stop.
if ( 0 == 1 ) then
cp -p Makefile.conf Makefile.conf.old
cp: cannot stat `Makefile.conf': No such file or directory
gmake SRCDIR=/blues/gpfs/home/bogensch/code/ACME_developments/ACME_BLHdevs_03202017_b/cime/externals/mct
Makefile:3: Makefile.conf: No such file or directory
gmake: *** No rule to make target `Makefile.conf'.  Stop.
exit 1

--- a/cime/cime_config/acme/machines/config_machines.xml
+++ b/cime/cime_config/acme/machines/config_machines.xml
@@ -998,7 +998,7 @@
     </module_system>
     <environment_variables>
       <env name="NETCDF_PATH">`which nc-config | xargs dirname | xargs dirname`</env>
-      <env name="PNETCDF_PATH">`which pnetcdf_version | xargs dirname | xargs dirname`</env>
+      <env name="PNETCDF_PATH" mpilib="!mpi-serial">`which pnetcdf_version | xargs dirname | xargs dirname`</env>
     </environment_variables>
     <environment_variables>
       <env name="OMP_STACKSIZE">256M</env>

